### ISSUE TITLE
master fsharpi crashes on startup

### ISSUE THREAD

**Issue by nosami** [fsharp/fsharp#769]
Please provide a succinct description of the issue.

#### Repro steps

Provide the steps required to reproduce the problem

1. Build and install fsharp/fsharp from master
2. run `fsharpi`

#### Expected behavior

F# Interactive launches

#### Actual behavior

```
WARNING: The Carbon driver has not been ported to 64bits, and very few parts of Windows.Forms will work properly, or at all
Stacktrace:

  at <unknown> <0xffffffff>
  at (wrapper managed-to-native) System.Windows.Forms.XplatUICarbon.CGDisplayBounds (intptr) [0x00002] in <7303396d94c24566ae56d1145b631b41>:0
  at System.Windows.Forms.XplatUICarbon.get_WorkingArea () [0x00005] in <7303396d94c24566ae56d1145b631b41>:0
  at System.Windows.Forms.XplatUICarbon.get_VirtualScreen () [0x00000] in <7303396d94c24566ae56d1145b631b41>:0
  at System.Windows.Forms.XplatUI.get_VirtualScreen () [0x00000] in <7303396d94c24566ae56d1145b631b41>:0
  at System.Windows.Forms.Screen..cctor () [0x00034] in <7303396d94c24566ae56d1145b631b41>:0
  at (wrapper runtime-invoke) object.runtime_invoke_void (object,intptr,intptr,intptr) [0x0001e] in <d3a5f406e2b94a64ae3952c46c93e7a6>:0
  at <unknown> <0xffffffff>
  at System.Windows.Forms.Hwnd.GetNextStackedFormLocation (System.Windows.Forms.CreateParams,System.Windows.Forms.Hwnd) [0x00064] in <7303396d94c24566ae56d1145b631b41>:0
  at System.Windows.Forms.XplatUICarbon.CreateWindow (System.Windows.Forms.CreateParams) [0x000c5] in <7303396d94c24566ae56d1145b631b41>:0
  at System.Windows.Forms.XplatUI.CreateWindow (System.Windows.Forms.CreateParams) [0x00000] in <7303396d94c24566ae56d1145b631b41>:0
  at System.Windows.Forms.NativeWindow.CreateHandle (System.Windows.Forms.CreateParams) [0x00009] in <7303396d94c24566ae56d1145b631b41>:0
  at System.Windows.Forms.Control.CreateHandle () [0x00031] in <7303396d94c24566ae56d1145b631b41>:0
  at System.Windows.Forms.Form.CreateHandle () [0x00000] in <7303396d94c24566ae56d1145b631b41>:0
  at Sample.Microsoft.FSharp.Compiler.Interactive.Main/DummyForm.DoCreateHandle () [0x00001] in <597b21f6046a099ca7450383f6217b59>:0
  at (wrapper remoting-invoke-with-check) Sample.Microsoft.FSharp.Compiler.Interactive.Main/DummyForm.DoCreateHandle () [0x00031] in <597b21f6046a099ca7450383f6217b59>:0
  at Sample.Microsoft.FSharp.Compiler.Interactive.Main/WinFormsEventLoop..ctor () [0x00014] in <597b21f6046a099ca7450383f6217b59>:0
  at Sample.Microsoft.FSharp.Compiler.Interactive.Main/fsiWinFormsLoop@228.Invoke (Microsoft.FSharp.Core.Unit) [0x00002] in <597b21f6046a099ca7450383f6217b59>:0
  at Microsoft.FSharp.Control.LazyExtensions/Create@6356<T_REF>.Invoke () [0x00001] in <597b2272dff9fae1a745038372227b59>:0
  at System.Lazy`1<T_REF>.CreateValue () [0x00060] in <d3a5f406e2b94a64ae3952c46c93e7a6>:0
  at System.Lazy`1<T_REF>.LazyInitValue () [0x00080] in <d3a5f406e2b94a64ae3952c46c93e7a6>:0
  at System.Lazy`1<T_REF>.get_Value () [0x0003a] in <d3a5f406e2b94a64ae3952c46c93e7a6>:0
  at Sample.Microsoft.FSharp.Compiler.Interactive.Main/evaluateSession@240-3.EventLoopRun () [0x00001] in <597b21f6046a099ca7450383f6217b59>:0
  at Microsoft.FSharp.Compiler.Interactive.Shell.runLoop@2389 (Microsoft.FSharp.Compiler.Interactive.Shell/FsiConsoleOutput,Microsoft.FSharp.Compiler.Interactive.Shell/FsiEvaluationSessionHostConfig,Microsoft.FSharp.Core.Unit) [0x0004e] in <597b224cf1806346a74503834c227b59>:0
  at Microsoft.FSharp.Compiler.Interactive.Shell.DriveFsiEventLoop (Microsoft.FSharp.Compiler.Interactive.Shell/FsiEvaluationSessionHostConfig,Microsoft.FSharp.Compiler.Interactive.Shell/FsiConsoleOutput) [0x00005] in <597b224cf1806346a74503834c227b59>:0
  at Microsoft.FSharp.Compiler.Interactive.Shell/FsiEvaluationSession.Run () [0x00151] in <597b224cf1806346a74503834c227b59>:0
  at Sample.Microsoft.FSharp.Compiler.Interactive.Main.evaluateSession (string[]) [0x001da] in <597b21f6046a099ca7450383f6217b59>:0
  at Sample.Microsoft.FSharp.Compiler.Interactive.Main.MainMain (string[]) [0x0006a] in <597b21f6046a099ca7450383f6217b59>:0
```

#### Known workarounds

None

#### Related information 

Provide any related information 

On OSX, Mono 5.4.0.73

**Comment by dsyme** [comment]
@nosami This should be fixed by #767 - could you confirm please?

**Comment by nosami** [comment]
I just tested HEAD 9687f27c3e6be7b9e1646bb9ee1ae0b02080daae. It's working again, thanks!
